sgx_mm.c

```diff
@@ -445,7 +445,7 @@ int sgx_mm_enclave_pfhandler(const sgx_pfinfo* pfinfo)
     return ret;
 }
 
-int sgx_mm_init(size_t user_base, size_t user_end)
+int sgx_mm_init(size_t user_base, size_t user_end, size_t init_reserve_size)
```

Comment:
We may also need to update the related docs and comments, e.g.,
https://github.com/intel/sgx-emm/blob/222b9d8cd246809cc8da041241824b0d0469000f/design_docs/SGX_EMM.md?plain=1#L572-L590
https://github.com/intel/sgx-emm/blob/222b9d8cd246809cc8da041241824b0d0469000f/README.md?plain=1#L117
https://github.com/intel/sgx-emm/blob/98ac79b4460f3ee4c1c6d18126a607db2b1d111d/include/emm_private.h#L45-L60

BTW, this change seems to break API compatibility (I'm not sure if `sgx_mm_init()` is actually a private or a public API, though it's placed in `emm_private.h` but seems to be needed anyway?). Is this sth. we'd expect?